android/res/layout/fragment_search.xml

```diff
@@ -44,12 +44,12 @@
 
     <com.frostwire.android.gui.views.RichNotification
             android:id="@+id/fragment_search_rating_reminder_notification"
-            frostwire:rich_notification_icon="@null"
+            frostwire:rich_notification_icon="@drawable/rate_app_notification"
             frostwire:rich_notification_title="@string/rate_our_app"
             frostwire:rich_notification_title_underlined="true"
             frostwire:rich_notification_description="@string/rating_reminder_description"
             frostwire:rich_notification_number_of_action_links="2"
-            frostwire:rich_notification_action_links_horizontal_margin="14"
+            frostwire:rich_notification_action_links_horizontal_margin="25"
```

Comment:
is this `dp`?

Reply:
These are coded in RichNotification.java but it was not me and I'm not entirely sure.